vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -42,8 +42,8 @@
 open System
 
 
-/// Modules are the primary way to organize functions and values in F#. This module contains some
-/// basic values involving basic numeric values computed in a few different ways.
+/// A module is a grouping of F# code, such as values, types, and function values, in an F# program. 
```

Comment:
Nit: I'd remove "in an F# program."  It's already stated at the beginning of the sentence that we're dealing with F# code.